package.json

```diff
@@ -98,5 +100,6 @@
   },
   "ava": {
     "verbose": true
-  }
+  },
+  "typings": "./out/electron-builder.d.ts"
```

Comment:
Simplify programmatic usage — https://github.com/Microsoft/TypeScript/wiki/Typings-for-npm-packages